src/vue/backend/src/store/modules/general/actions.js

```diff
@@ -7,6 +7,11 @@ const generalSettingsStateChange = ({ commit }, payload) => {
     })
 }
 
+const promiseWithTimeout = () => {
```

Comment:
This is supposed to be a placeholder for the action that will be committed, and should be named accordingly. Additionally, add a `To-Do` comment in there so we know where to connect the api